docs/docs/jobs.md

```diff
@@ -100,6 +100,7 @@ print('Status: %s' % job.get_status())
 
 Some interesting job attributes include:
 * `job.get_status()` Possible values are `queued`, `started`, `deferred`, `finished`, `stopped`, `scheduled` and `failed`
+* `job.get_meta()` Returns custom `job.meta` dict containing user stored data
```

Comment:
Can we add the `refresh` argument here? Also we should clarify that this will read `meta` directly from Redis if `refresh=True`

Reply:
Done